src/vmm/src/lib.rs

```diff
@@ -46,13 +46,18 @@ use crate::device_manager::mmio::MMIODeviceManager;
 use crate::memory_snapshot::SnapshotMemory;
 #[cfg(target_arch = "x86_64")]
 use crate::persist::{MicrovmState, MicrovmStateError, VmInfo};
+use crate::rpc_interface::{ActionResult, VmmActionError, VmmData};
```

Comment:
`rpc_interface` is one layer above the `vmm` (for now, only conceptually, later we should actually enforce it by taking it out of the `Vmm` crate).

This file `vmm/src/lib.rs` is strictly _vmm programmatic interface_, it should not have any knowledge (aka dependencies) to upper layers such as the `rpc_interface`.

Don't use `rpc_interface` here.